### ISSUE TITLE
Ban cjs & cts file extensions

### ISSUE THREAD

**Issue by dsherret** [jsr-io/jsr#713]
These should be banned as they don't work.

**Comment by mbhrznr** [comment]
how would such banning look like?
i'd assume it could be a thrown error during publish/dry run?

i'd love to give it a spin, as it might be a nice opportunity to check the inner workings of jsr.

**Comment by crowlKats** [comment]
>  i'd assume it could be a thrown error during publish/dry run?

correct; and i think then on JSR's backend we should handle it as well in case
